pkg/cluster/cluster.go

```diff
@@ -576,6 +576,14 @@ func (c *Cluster) Update(oldSpec, newSpec *acidv1.Postgresql) error {
 				c.logger.Errorf("could not sync statefulsets: %v", err)
 				updateFailed = true
 			}
+		} else {
+			// patroni configuration is not always reflected on the StatefulSet as they have
+			// different lifecycle (due to SPILO's implementation - see: issues#501) but they
+			// still need to be updated via the Patroni REST API.
+			if err := c.applyPatroniConfig(); err != nil {
```

Comment:
is the intention here to update Patroni config in DCS on every `Update` event even if the actual config did not change ?